ksql-engine/src/main/java/io/confluent/ksql/util/ExpressionTypeManager.java

```diff
@@ -193,14 +194,27 @@ protected Expression visitFunctionCall(final FunctionCall node,
 
     final UdfFactory udfFactory = functionRegistry.getUdfFactory(node.getName().getSuffix());
     if (udfFactory != null) {
-      List<Schema.Type> argTypes = new ArrayList<>();
-      for (final Expression expression : node.getArguments()) {
-        process(expression, expressionTypeContext);
-        argTypes.add(expressionTypeContext.getSchema().type());
+      if (node.getName().getSuffix().equalsIgnoreCase(FetchFieldFromStruct.functionName)) {
```

Comment:
`getUdfFactory` should throw the exception, rather than return null.

Reply:
Again, is there any way to avoid this special casing for this function?

Reply:
getUDFFactory related to the UDF work, this PR just uses it.
For special case please refer to the above comment.